CONTRIBUTING.md

```diff
@@ -31,16 +33,14 @@ pub fn my_algorithm() {
 #[cfg(test)]
 mod tests {
     #[test]
-    fn my_algorithm() {
+    fn my_test() {
         // ...
     }
 }
 ```
 
 ## Before submitting you PR
 
-Make sure you ran:
-  * `cargo fmt`
-  * `cargo test`
-
-and squashed your branch !
+Make sure you:
+  * ran `cargo fmt` and `cargo test`
+  * **squashed** your branch !
```

Comment:
we don't require squashing, it's extra burdon on the contributors and it's tricky for people new to git.
Moreover, we have an option to squash and merge when we merge the PR.

Reply:
Alright alright !